icu4c/source/i18n/unicode/basictz.h

```diff
@@ -49,7 +50,7 @@ class U_I18N_API BasicTimeZone: public TimeZone {
      * @return clone, or nullptr if an error occurred
      * @stable ICU 3.8
      */
-    virtual BasicTimeZone* clone() const = 0;
+    virtual BasicTimeZone* clone() const override { UPRV_UNREACHABLE; return nullptr; }
```

Comment:
This change a build/compile time error/assertion to a run time assertion/error and I am concern about this.

Reply:
virtual BasicTimeZone* clone() const override = 0;